hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/file/OMFileCreateResponse.java

```diff
@@ -18,24 +18,25 @@
 
 package org.apache.hadoop.ozone.om.response.file;
 
-import javax.annotation.Nullable;
 import javax.annotation.Nonnull;
 
 import org.apache.hadoop.ozone.om.helpers.OmKeyInfo;
 import org.apache.hadoop.ozone.om.response.key.OMKeyCreateResponse;
 import org.apache.hadoop.ozone.protocol.proto.OzoneManagerProtocolProtos
     .OMResponse;
 
-
-
 /**
  * Response for crate file request.
  */
 public class OMFileCreateResponse extends OMKeyCreateResponse {
 
-  public OMFileCreateResponse(@Nullable OmKeyInfo omKeyInfo,
-      long openKeySessionID, @Nonnull OMResponse omResponse) {
-    super(omKeyInfo, openKeySessionID, omResponse);
+  public OMFileCreateResponse(@Nonnull OMResponse omResponse,
+      @Nonnull OmKeyInfo omKeyInfo, long openKeySessionID) {
+    super(omResponse, omKeyInfo, openKeySessionID);
+  }
+
+  public OMFileCreateResponse(@Nonnull OMResponse omResponse) {
+    super(omResponse);
```

Comment:
Minor: Missing checkStatusNotOK() and also can we add similar javadoc as OMKeyCreateResponse.

Reply:
Thanks for catching this. If you are ok with rest of the patch, I add this check in the next Jira HDDS-2944.

Reply:
Sure.